acinclude/win32-sspi.m4

```diff
@@ -0,0 +1,40 @@
+## Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+##
+## Squid software is distributed under GPLv2+ license and includes
+## contributions from numerous individuals and organizations.
+## Please see the COPYING and CONTRIBUTORS files for details.
+##
+
+# check if the Windows SSPI requirements are available and work.
+# first argument is the variable containing the result
+#   (will be set to "yes" or "no")
+# second argument is the commands to run on success
+#
+AC_DEFUN([SQUID_CHECK_WIN32_SSPI],[
+  AC_CACHE_CHECK([whether Windows SSPI is available],[$1],[
```

Comment:
PR build [log](https://github.com/squid-cache/squid/suites/13813788092/artifacts/766560267):

```
checking whether Windows SSPI is available... checking for w32api/windows.h... no
checking for windows.h... (cached) no

configure: helper auth/basic/SSPI ... found but cannot be built
```

I see a few problems in the above output (but please ignore any specifics that are inaccurate -- no need to argue about these low-level details):

1. The first line appears to contain what should be two separate lines.
2. The outcome for "checking whether Windows SSPI is available..." (or "checking for w32api/windows.h...") check is not visible.
3. There is an empty line after "checking for windows.h... (cached) no"

Official build [log](https://github.com/squid-cache/squid/suites/13796614278/artifacts/765425688) looks much better:

```
checking for w32api/windows.h... no
checking for windows.h... (cached) no
configure: helper auth/basic/SSPI ... found but cannot be built
```

Please fix and check for other problems by comparing "before" and "after" ./configure output.

Reply:
Fixed in [de3c084](https://github.com/squid-cache/squid/pull/1348/commits/de3c0845687cb525af643f1b55fec2870180ed3a)

Output now looks like:
```
checking for w32api/windows.h... no
checking for windows.h... (cached) yes
checking for wchar.h... (cached) yes
checking for tchar.h... yes
checking for ntsecapi.hhh... no
checking for security.h... yes
checking for sspi.h... yes
configure: helper auth/basic/SSPI ... found but cannot be built
```